### PR TITLE
Watch later

### PR DESCRIPTION
**Pull request by lannisterrr**
Added - Implemented add to watch later functionality
review - first 10 files

### CODE REVIEW

File: src/Pages/WatchLaterPage.js [hunk 1]
```diff
@@ -1,7 +1,51 @@
...
+          },
+        });
+        dispatch({ type: 'GET_WATCHLATER', payload: res.data.watchlater });
+      } catch (error) {
+        console.log(error);
+      }
+    })();
+  }, []);
+
+  console.log(dataState.watchLater);
```
Comment (shivampandey0) [inline src/Pages/WatchLaterPage.js:27]: remove `console.log() `before merging

**Review by shivampandey0 — APPROVED** [review]
- Good folder structure
- Semantic HTML used
- Structured CSS

Good to merge

File: src/components/ListingVideoComponent.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ function ListingVideoComponent({ video, children }) {
           <img src={thumbnail.url} alt={thumbnail.altText} />
           <figcaption>
             <span>
-              <i class="fa-solid fa-play f-8 t-c-1"></i>
+              <i className="fa-solid fa-play f-8 t-c-1"></i>
             </span>
             <span className="f-8 f-bold t-c-1">play</span>
           </figcaption>
```
Comment (ashishnnnnn) [inline src/components/ListingVideoComponent.js:25]: Liked the use of semantic html..

**Review by ashishnnnnn — APPROVED** [review]
Code can be merged ..
Good Work :+1:

**Pull request merged**